### PR TITLE
15971 - fix ugly looking expiry date

### PR DESCRIPTION
**Pull request by jonathan-longe**
*Issue #:* /bcgov/entity#15971

Instead of this hot mess:
image

We have this:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue:32]: According to uxpin, this should be the long version ("August" not "Aug").

https://preview.uxpin.com/f3e9c0da57601f193d2f0c6b1946134b2ddeb2fe#/pages/160694550/comment/sitemap

File: tests/unit/LedgerConsentContinuationOut.spec.ts
Comment (severinbeauvais) [inline tests/unit/LedgerConsentContinuationOut.spec.ts:55]: Thanks for this!

File: src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue
Comment (severinbeauvais) [reply src/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue]: Beauty.

**Review by severinbeauvais — APPROVED** [review]
Please wait for another review or 2 before merging.

**Review by ketaki-deodhar — APPROVED** [review]
Looks good! Thanks @jonathan-longe

**Review by JazzarKarim — APPROVED** [review]
LGTM!

File: tests/unit/LedgerConsentContinuationOut.spec.ts
Comment (jonathan-longe) [reply tests/unit/LedgerConsentContinuationOut.spec.ts:55]: Good catch @severinbeauvais .  Updated to use full month name.

**Pull request merged**
